pages/index.tsx

```diff
@@ -23,24 +23,41 @@ import { CreateListProvider, useCreateList, usePagination } from 'hooks'
 import { Resource, RESOURCE_SEARCH_FIELDS } from 'models'
 import { NextPage } from 'next'
 import Head from 'next/head'
-import { useEffect, useMemo, useState } from 'react'
+import { useMemo, useState } from 'react'
 import { Search } from 'react-feather'
 import { useQuery } from 'react-query'
-import { fuseSearch, onEnter } from 'utils'
+import Select from 'react-select'
+import { fuseSearch } from 'utils'
 
 export const HomePage: NextPage = () => {
-  const { data: allResources } = useQuery<Resource[], Error>(
-    ['allResources'],
-    () => getAllResources()
+  const { data: allResources } = useQuery('getAllResources', () =>
+    getAllResources()
```

Comment:
We're not using the error type so just let `useQuery` infer the type of `data`

Reply:
Do we lose anything from being explicit with our types? Either way though if it can infer the type that should be fine

Reply:
Not really - if we explicitly typed, it would just give us an error if the the explicit type is different from the return type of `getAllResources()` which I don't think would be that helpful? 

If you think it you would be though I can add it back - I just think it looks a bit cleaner this way.

Reply:
Nope thats all good I was just curious